dockstore-webservice/src/main/resources/migrations.1.12.0.xml

```diff
@@ -102,4 +102,28 @@
             create unique index collection_displayname_index on collection (LOWER(displayname), organizationid);
         </sql>
     </changeSet>
+
+    <changeSet author="svonworl" id="categories_new_feature">
+        <addColumn tableName="organization">
+            <column name="categorizer" type="boolean" defaultValue="false">
+                <constraints nullable="false"/>
+            </column>
+        </addColumn>
+        <!-- the following addColumn+addNotNullConstraint combo is necessary because the JPA-generated db wants the column "collection" to be non-null, but complains if there is an SQL-defined default value. -->
+        <addColumn tableName="collection">
+            <column name="dtype" type="varchar(31)" value="Collection">
+            </column>
+        </addColumn>
+        <addNotNullConstraint columnName="dtype" tableName="collection"/>
+        <sql dbms="postgresql">
+            create unique index collection_categoryname_index on collection (LOWER(name)) where dtype = 'Category';
+        </sql>
+        <sql dbms="postgresql">
+            insert into organization (name, displayname, description, status, categorizer, link, email, dbcreatedate, dbupdatedate) values ('dockstore', 'Dockstore', 'Each of this organization''s Collections is a Category.', 'HIDDEN', true, 'https://dockstore.org/', 'support@dockstore.org', LOCALTIMESTAMP, LOCALTIMESTAMP);
+        </sql>
+        <!-- Add a handful of key users to the dockstore org, rather than all admins, to avoid inadvertently binding test users to orgs, which interferes with existing integration tests. -->
+        <sql dbms="postgresql">
+            insert into organization_user (organizationid, userid, accepted, role) select (select id from organization where name = 'dockstore'), id, true, 'ADMIN' from enduser where username in ('dockstore-admin', 'gluu-oicr', 'coverbeck-adm');
```

Comment:
Feels a little weird identifying the names of specific admin accounts. This doesn't seem right. 
Maybe you can migrate admins that are not the test users.

Reply:
Yeah, I can add admins and exclude the test admins, that'll work.